metadata.json

```diff
@@ -120,5 +120,5 @@
     "vmware-tools",
     "vmware_tools"
   ],
-  "version": "2.1.1-rc0"
+  "version": "3.0.1-rc0"
```

Comment:
please go to 3.0.0

Reply:
This was added by the rake task as separate commit, after tagging it...
```
* 2b069c7 Christoph Maser 37 minutes ago  (tag: v3.0.0)  Release 3.0.0
* 2b069c7 Christoph Maser 37 minutes ago  (tag: v3.0.0)  Release 3.0.0
```
I just dropped the extra commit